cmd/dependabot/internal/cmd/update_test.go

```diff
@@ -12,6 +12,10 @@ import (
 )
 
 func Test_processInput(t *testing.T) {
+	t.Cleanup(func() {
+		os.Unsetenv("LOCAL_GITHUB_ACCESS_TOKEN")
+		os.Unsetenv("LOCAL_GITHUB_ACCESS_TOKEN")
```

Comment:
Did this need to be unset twice? Mostly just curious.

Reply:
Copy-paste 🤦 The 2nd one was supposed to be `LOCAL_AZURE_ACCESS_TOKEN`.